libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/MultiLanguageRecognizer.cs

```diff
@@ -46,7 +46,7 @@ public MultiLanguageRecognizer([CallerFilePath] string callerPath = "", [CallerL
         public override async Task<RecognizerResult> RecognizeAsync(DialogContext dialogContext, Activity activity, CancellationToken cancellationToken = default, Dictionary<string, string> telemetryProperties = null, Dictionary<string, double> telemetryMetrics = null)
         {
             var policy = new List<string>();
-            if (LanguagePolicy.TryGetValue(activity.Locale, out string[] targetpolicy))
+            if (activity.Locale != null && LanguagePolicy.TryGetValue(activity.Locale, out string[] targetpolicy))
```

Comment:
>Locale [](start = 79, length = 6)

you should do 
  if (LanguagePolicy.TryGetValue(activity.Locale ?? String.Empty, out string[] targetpolicy))

We use "" as "Default language policy"